### DISCUSSION TITLE
Does .nomedia exclude subfolders?

### DISCUSSION THREAD

**Discussion by Schema-F** [nextcloud/recognize#619] [Q&A] [answered]
I'm a new recognize user and have a question i didn't yet find an answer for: When i use .nomedia to exclude a folder from being scanned, do i need to repeat the .nomedia tag in all subfolders to exclude them, too?

And: Is the appdata*/previews folder scanned by default?

**Comment by marcelklehr** [comment, marked as answer]
appdata should not be scanned. .nomedia files exclude a folder with all its subfolders

**Reply by Schema-F** [reply]
Thank you for your fast response and your great work!
